core/tx_pool_test.go

```diff
@@ -1275,13 +1275,13 @@ func TestTransactionPoolRepricingKeepsLocals(t *testing.T) {
 	// Create transaction (both pending and queued) with a linearly growing gasprice
 	for i := uint64(0); i < 500; i++ {
 		// Add pending
-		p_tx := pricedTransaction(i, 100000, big.NewInt(int64(i)), keys[2])
-		if err := pool.AddLocal(p_tx); err != nil {
+		pTx := pricedTransaction(i, 100000, big.NewInt(int64(i)), keys[2])
```

Comment:
Lets name this `pending`

Reply:
Done. But note that `pending` is a variable name used outside the scope of this for loop on line 1283.  An alternative could be `pendingTx`?

Reply:
Sure, I'm just not a fan of p, q :)

Reply:
But we're both fans of copy-paste!